src/main/java/com/han95210/codesquad/monsterRace/step2/MonsterRace.java

```diff
@@ -0,0 +1,68 @@
+package com.han95210.codesquad.monsterRace.step2;
+
+import java.util.Scanner;
+
+public class MonsterRace {
+
+    private Scanner scanner;
+
+    private int getNumberOfMonster() {
+        scanner = new Scanner(System.in);
+
+        // 몬스터 수 정하기
+        System.out.println("몬스터는 모두 몇 마리인가요?");
+        return scanner.nextInt();
+    }
+
+    // 기능 1개 : 시도할 횟수 입력 받기
+    private int getNumberAttempts() {
+        scanner = new Scanner(System.in);
+
+        // 시도 횟수 정하기
+        System.out.println("시도할 횟수는 몇 회 인가요?");
+        return scanner.nextInt();
+    }
+
+    private static String getRandomDash() {
+        String dashStr = "";
+        int randomNum = (int) (Math.random() * 10);
+        if (randomNum >= 4) {
+            dashStr = "-";
+        }
+        return dashStr;
+    }
```

Comment:
코드 내에서 `(int) (Math.random() * 10);` 라는 코드가 두번 정도 호출되어지고 있네요.

이 부분은 private method로 추출해서 사용해도 될 것 같아요.

```java
public Integer getRandomNumber(int range) {
    return (int) (Math.random() * range);
}
```

```suggestion
    private String getRandomDash() {
        if (getRandomNum(10) >= 4) {
             return "-";
        }
        return "";
    }
```